pkgs/tools/filesystems/sandboxfs/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, fuse
+, pkg-config
+, installShellFiles
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sandboxfs";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "bazelbuild";
+    repo = pname;
+    rev = "sandboxfs-${version}";
+    sha256 = "Ia6rq6FN4abnvLXjlQh4Q+8ra5JThKnC86UXC7s9//U=";
+  };
+
+  cargoSha256 = "sha256-k303TjWG+n+/ZMmko84KJtYb7swuQ1ZJOc4Vq6aOhX0=";
+
+  cargoPatches = [ ./Cargo.lock.patch ];
+
+  buildInputs = [ fuse ];
+  nativeBuildInputs = [ pkg-config installShellFiles ];
```

Nitpick, but this is the canonical order ;).

```suggestion
  nativeBuildInputs = [ pkg-config installShellFiles ];

  buildInputs = [ fuse ];
```

---

Done.